res/values/strings.xml

```diff
@@ -740,6 +768,8 @@
     <string name="global_menu_help_contribute_desktop">Contribute on GitHub</string>
     <string name="global_menu_help_report_desktop">Report an issue</string>
     <string name="global_menu_help_about_desktop">About Delta Chat</string>
+    <string name="global_menu_file_open_desktop">Open DeltaChat</string>
+    <string name="global_menu_minimize_to_tray">Minimize</string>
```

Comment:
if the wording is really just "minimize", i would also name the key as such.

Reply:
IDK I think this could also change in the future :shrug: 
CC @Jikstra

Reply:
k, then maybe leave that as is.